packages/cloud_firestore/cloud_firestore/pubspec.yaml

```
@@ -6,26 +6,26 @@ homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/c
 version: 0.16.0
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
-  flutter: ">=1.12.13+hotfix.4 <2.0.0"
+  sdk: ">=2.12.0-133.7.beta <3.0.0"
```

```suggestion
  sdk: ">=2.12.0-0 <3.0.0"
```

This is fine as it was